config/config.yml

```diff
@@ -6,6 +6,7 @@ defaults: &defaults
   plum_solr_url: <%= "http://127.0.0.1:#{ENV['TEST_JETTY_PORT'] || 8984}/solr/blacklight-core-test" %>
   plum_binary_path: "/mnt/diglibdata/hydra_binaries"
   plum_derivative_path: "/mnt/libimages1/data/jp2s/plum_prod"
+  bag_path: <%= Rails.root.join("tmp", "bags") %>
```

Comment:
I assume we'll setup an ENV variable in production to put these somewhere outside of `tmp`

Reply:
Done.